TankDriver.App/Logic/BulletSpace.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using TankDriver.Models;
+
+namespace TankDriver.Logic
+{
+	internal class BulletSpace: IUnit
+	{
+		public List<Bullet> Bullets { get; private set; }
```

Comment:
In case we'll let this class live, you need to be cool (i.e. C# 6) here: 
```csharp
    public List<Bullet> Bullets { get; } = new List<Bullet>();
```